modules/programs/rtorrent.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.rtorrent;
+
+in
+
+{
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.rtorrent = {
+    enable = mkEnableOption "rTorrent";
+
+    config = mkOption {
```

Comment:
I would recommend calling this option "settings" instead of the already overloaded word "config".

Reply:
makes sense, fixed